app/scripts/metamask-controller.js

```diff
@@ -302,8 +309,19 @@ export default class MetamaskController extends EventEmitter {
     this.createVaultMutex = new Mutex();
 
     this.extension.runtime.onInstalled.addListener((details) => {
-      if (details.reason === 'update' && version === '8.1.0') {
-        this.platform.openExtensionInBrowser();
+      if (details.reason === 'update') {
+        if (version === '8.1.0') {
+          this.platform.openExtensionInBrowser();
+        }
+        this.loggingController.add({
+          type: LogType.GenericLog,
+          data: {
+            event: 'Extension version update',
```

Comment:
Should we create a constant for this in case we have more in future? Something like `LOG_EVENT`?

Reply:
Added in https://github.com/MetaMask/metamask-extension/pull/21077/commits/e5c4a055e60851842129eac7ca94fec18794d228